nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -54,6 +54,6 @@ package object runtime {
   def init(argc: Int, argv: Ptr[Ptr[Byte]]): ObjectArray = {
     GC.init()
     null
-  }
+  }  
```

Comment:
@olafurpg Any idea why scalafmtTest doesn't complain about this trailing whitespace?